tests/test_algorithm/test_get_conditional_independencies.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+
+"""Test getting conditional independencies."""
+
+import unittest
+from itertools import chain
+from typing import Set
+
+from ananke.graphs import ADMG
+
+from y0.algorithm import falsification
+from y0.algorithm.conditional_independencies import ConditionalIndependency, get_conditional_independencies
+from y0.examples import examples
+from y0.graph import NxMixedGraph
+
+
+class TestDSeparation(unittest.TestCase):
+    "Test the d-separation utility."
+
+    # TODO: Migrate to the ADMG representation, not just vanilla networkx
+    def test_mit_example(self):
```

Comment:
it would be great to curate examples from the example graphs in `y0.examples`

Reply:
I can do that.